client/src/index.tsx

```diff
@@ -7,12 +7,14 @@ import { Integrations } from '@sentry/tracing'
 import './index.css'
 import App from './App'
 
-Sentry.init({
-  dsn: (window as any)._arlo_sentry_dsn, // eslint-disable-line @typescript-eslint/no-explicit-any
-  environment: (window as any)._arlo_flask_env, // eslint-disable-line @typescript-eslint/no-explicit-any
-  integrations: [new Integrations.BrowserTracing()],
-  tracesSampleRate: 0.2,
-})
+if (process.env.NODE_ENV === 'production') {
+  Sentry.init({
+    dsn: (window as any)._arlo_sentry_dsn, // eslint-disable-line @typescript-eslint/no-explicit-any
+    environment: (window as any)._arlo_flask_env, // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
I know these were already this way before, but you can extend the `window` interface to include additional properties a la [`window.kiosk`](https://github.com/votingworks/vxsuite/blob/d40343e36e44292768642accd8ebdaeec5d04e4b/libs/%40types/kiosk-browser/kiosk-browser.d.ts#L263).

Reply:
Oh cool, it's about time I learn how to use declaration files